docs-next/components/Heading.tsx

```diff
@@ -0,0 +1,128 @@
+/** @jsx jsx */
+import { useRef } from 'react';
+import { jsx } from '@keystone-ui/core';
+import slugify from '@sindresorhus/slugify';
+import { CopyToClipboard } from './CopyToClipboard';
+
+// offset header height for hash links
+const hashLinkOffset = {
+  '::before': {
+    content: '" "',
+    height: 'calc(32px + 60px)',
+    display: 'block',
+    marginTop: -60,
+  },
+};
+const getAnchor = (text: string) => {
+  if (typeof text === 'string') {
+    return slugify(text);
+  }
+  return '';
+};
+
+interface StringOnlyChildren {
+  children: string;
+}
+
+interface HeadingProps extends StringOnlyChildren {
+  as: 'h1' | 'h2' | 'h3' | 'h4' | 'h5' | 'h6';
+  css: Record<string, any>;
```

Comment:
@mitchellhamilton  not really sure what to do here besides import the CSSObject. 
Let me know if there's a better way to infer this type in this particular use case

Reply:
It shouldn't be there, className should be there(and it should be optional). the jsx pragma automatically adds the css prop if a component accepts className.

Reply:
Perfect, ty!